infrastructure/evault-provisioner/src/templates/evault.nomad.ts

```
@@ -14,118 +17,94 @@ export function generatePassword(length = 16): string {
     return result;
 }
 
-export function generateNomadJob(w3id: string, eVaultId: string) {
-    const neo4jUser = "neo4j";
-    const neo4jPassword = generatePassword(24);
+export async function provisionEVault(w3id: string, eVaultId: string) {
+    const idParts = w3id.split('@');
+    w3id = idParts[idParts.length - 1]
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)